.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
+  "env": { "browser": true, "es2021": true },
+  "extends": "eslint:recommended",
+  "overrides": [
+    {
+      "files": ["**/*.test.js"],
+      "env": { "jest": true, "cypress/globals": true},
```

Comment:
You should have two separate overrides for jest and cypress instead of merging them together.